esp/services/ws_esdlconfig/ws_esdlconfigservice.cpp

```diff
@@ -186,8 +186,11 @@ void CWsESDLConfigEx::init(IPropertyTree *cfg, const char *process, const char *
     if(servicecfg == NULL)
         throw MakeStringException(-1, "config not found for service %s/%s",process, service);
 
-    m_isDetachedFromDali = false;
     m_esdlStore.setown(createEsdlCentralStore());
+    if(nullptr == m_esdlStore)
```

Comment:
Missing space after the if.

Btw, trivial, and no need to change but I see below you did:

`m_isDetachedFromDali = (nullptr == m_esdlStore);
`

You could do the same here.

Reply:
Made that change as part of the final squash.